src/components/grid-aware/ThreeParagraphBlock/ThreeParagraphBlock.module.css

```diff
@@ -148,22 +126,19 @@
     margin-top: 30px;
   }
 
-  .ctaTitleBlock {
-    margin-top: 40px;
-    max-width: 100%;
+  .paragraph3Wrapper {
+    margin-top: 30px;
   }
 
   .image1Wrapper {
     display: none;
   }
 
   .image2Wrapper {
-    display: none;
-  }
-
-  .image3Wrapper {
+    /* overflow: hidden; */
```

Comment:
Let's delete commented out code instead of committing it. See https://kentcdodds.com/blog/please-dont-commit-commented-out-code for a short article describing why keeping commented out code in the codebase is not generally a good idea.